frontend/packages/kubevirt-plugin/integration-tests/views/vm.next.run.configuration.view.ts

```diff
@@ -0,0 +1,17 @@
+import { $, $$ } from 'protractor';
+
+export const isPCAlertPresent = async (): Promise<boolean> =>
```

Comment:
there is a quite overlap with pendingChangesInfoAlert and pendingChangesWarningAlert

Reply:
what is you point ? each has unique class that I'm using to identify it.

Reply:
that they were using the same class, but the constants file is gone now - so it is fine